docs/vmagent.md

```diff
@@ -1146,9 +1147,65 @@ If you have suggestions for improvements or have found a bug - please open an is
   its initialization for all the [service_discovery configs](https://docs.victoriametrics.com/sd_configs/).
   It may be useful to perform `vmagent` rolling update without any scrape loss.
 
+### Monitoring of monitoring using VictoriaMetrics Cloud
```

Comment:
I'm not sure these docs are needed here. It is enough to mention possibility to integrate with Cloud and #VictoriaMetrics-Cloud-integration section below. The rest of the docs should reside in Cloud docs.
wdyt @tenmozes ?